packages/checkout-button-integration/src/CheckoutButton.tsx

```diff
@@ -12,19 +12,49 @@ const CheckoutButton: FunctionComponent<CheckoutButtonProps> = ({
     methodId,
     onUnhandledError,
 }) => {
+    const beautifyCheckoutButtons = useCallback((): void => {
+        if (methodId === 'amazonpay') {
+            if (!document.querySelector('.checkout-button-container')) {
+                return;
+            }
+
+            const container: HTMLElement | null = document.querySelector(
```

Comment:
HTMLElement | undefined - double check it, please

Reply:
I checked it and it must stay HTMLElement | null as querySelector method is returning HTMLElement or null if it doesn't find any matches.

Reply:
I also modified a little bit code below so amazonPayButton variable can match HTMLElement | null types too (it was previously HTMLElement | null | undefined)